### DISCUSSION TITLE
Stable structure for this project

### DISCUSSION THREAD

**Discussion by thienphuoc** [axmolengine/axmol#1169] [Q&A] [answered]
Hi @halx99 I think we need a stable structure for this project. Because some people and I use it to develop games. So, sometimes we need an update to release head to fix some bugs on the engine. 

![Screenshot 2023-04-11 at 10 19 34](https://user-images.githubusercontent.com/17909394/231047840-bc940541-cda1-445d-9f97-37a644b3cd4e.png)

**Comment by halx99** [comment, marked as answer]
Now the structure should be stable.
